library/src/main/java/com/datatorrent/lib/transform/TransformOperator.java

```diff
@@ -56,12 +56,12 @@
 {
   @NotNull
   private Map<String, String> expressionMap = new HashMap<>();
-  private List<String> expressionFunctions = new LinkedList<>();
+  protected List<String> expressionFunctions = new LinkedList<>();
```

Comment:
expressionFunctions variable needs to be used in extended class. Hence the change.